From 69fce10d8a595ea90e4570fb9f559f95a509fbeb Mon Sep 17 00:00:00 2001
From: David Lyle <david.lyle@hp.com>
Date: Sat, 21 Jun 2014 14:29:04 -0600
Subject: [PATCH] Adding back the forms.Login import

Changing this breaks all older versions of Horizon. Adding it back
in to allow older versions to continue working.

Closes-Bug: #1332855
Change-Id: Icdb206d4095b1746eef9be02ca63f2aa9cfe1081
(cherry picked from commit 47fcf7ae4ad68b175914ab9611a59d359aa754c0)
---
 openstack_auth/views.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/openstack_auth/views.py b/openstack_auth/views.py
index e423b5a..d2c73e8 100644
--- a/openstack_auth/views.py
+++ b/openstack_auth/views.py
@@ -29,6 +29,10 @@ from keystoneclient import exceptions as keystone_exceptions
 from keystoneclient.v2_0 import client as keystone_client_v2
 
 from openstack_auth import forms
+# This is historic and is added back in to not break older versions of
+# Horizon, fix to Horizon to remove this requirement was committed in
+# Juno
+from openstack_auth.forms import Login  # noqa
 from openstack_auth import user as auth_user
 from openstack_auth import utils
 
